packages/zowe-explorer-api/src/tree/IZoweTreeNode.ts

```diff
@@ -37,32 +37,11 @@ export type ZosEncoding = TextEncoding | BinaryEncoding | OtherEncoding;
  * @export
  * @interface IZoweTreeNode
  */
-export interface IZoweTreeNode {
-    /**
-     * The icon path or [ThemeIcon](#ThemeIcon) for the tree item.
-     */
-    iconPath?: string | vscode.Uri | { light: string | vscode.Uri; dark: string | vscode.Uri } | vscode.ThemeIcon;
+export interface IZoweTreeNode extends vscode.TreeItem {
```

Comment:
Should we list all the properties removed from this interface in the changelog?
As well as those that just changed in their types due to the extension of the `vscodeTreeItem` 
For example:
- `tooltip: string | vscode.MarkdownString | undefined;` changed to `tooltip: string | vscode.MarkdownString;`

Reply:
I looked over the removed properties and added a changelog entry for the removal of the `encoding` property. Thank you for spotting that!

Regarding `tooltip` and other properties from `vscode.TreeItem`: Maybe I'm overlooking something, but the type signatures look to be the same for the ones I removed.